src/radical/pilot/agent/lm/jsrun.py

```diff
@@ -117,8 +116,15 @@ def construct_command(self, cu, launch_script_hop):
         if task_argstr: task_command = "%s %s" % (task_exec, task_argstr)
         else          : task_command = task_exec
 
-        env_list   = self.EXPORT_ENV_VARIABLES + task_env.keys()
-        env_string = ' '.join(['-E "%s"' % var for var in env_list])
+        env_string = ''
+      # task_env   = cud.get('environment') or dict()
+      # env_list   = self.EXPORT_ENV_VARIABLES + task_env.keys()
+      # env_string = ' '.join(['-E "%s"' % var for var in env_list])
+      #
+      # # jsrun fails if an -E export is not set
+      # for var in env_list:
+      #     if var not in os.environ:
+      #         os.environ[var] = ''
 
```

Comment:
This is commented out because `jsrun` does not fail anymore without -E?

Reply:
This is the counterpart to the comment in the other PR: here we disable `-E` completely in the lines before because it did not play well with some `module load` directives.  For now we rely on env settings in the unit shell script and on jsrun to export the right environment by default.  This may need more work later, but for now I would like to gather some data on how this behaves for actual workloads.